helm-chart/renku/Chart.yaml

```diff
@@ -3,4 +3,4 @@ appVersion: '1.0'
 description: A Helm chart for the Renku platform
 name: renku
 icon: https://github.com/SwissDataScienceCenter/renku-sphinx-theme/raw/master/renku_sphinx_theme/static/favicon.png
-version: 0.18.1-2.01781e6
+version: 0.23.1-5.5b17747.n048.h90d0df90
```

Comment:
I thought we never usually set this to anything in particular and let chartpress set it when we publish or install. For the longest time it used to be set to `0.0.1-set.by.chartpress`. I think we should revert to something like that or to `latest` or to the latest published version.